docs/faq.rst

```diff
@@ -73,6 +73,19 @@ The VLAN encap values associated with logical interfaces of the LDEV change and
 
 After a VLAN tag is updated on BIG-IP, the visibility vlan table will start showing the VLANs again.
 
+
+**Q. Why don't I see all the VLANs/VIPs/Nodes from the BIG-IP in the visibility tables?**
+
+Visibility tables display only those entries from BIG-IP which have corresponding constructs on APIC. For example a VLAN from BIG-IP will only be displayed if that VLAN also belongs to some Tenant|App Profile|EPG or Tenant|LDEV on APIC. Similarly, a node will only be displayed, if it exists as an operational endpoint in one or more of the EPGs on APIC. 
+
+
+**Q. In Visibility tables, why don't I see Common partition entries in VLAN/VIP/Node table when I select a different partition?**
+
+The F5 ACI ServiceCenter Visibility tables have an option to select the **Partition**. The VLAN/VIP/Node tables will only display entries from the selected partition and will not include Common partition entries (Although, BIG-IP UI does provide this feature where any partition selected will also show entries from Common partition). 
```

Comment:
"entries (although the BIG-IP UI does... 
...will also show entries from the Common partition)."